service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -229,12 +272,38 @@ public Deployable definition(DefinitionContext definitionContext) {
               .withRegion(resourceGroup.region())
               .withExistingResourceGroup(resourceGroup.name());
 
+      var storageAuditLogSettings =
+          armManagers
+              .monitorManager()
+              .diagnosticSettings()
+              .define(
+                  nameGenerator.nextName(ResourceNameGenerator.MAX_DIAGNOSTIC_SETTING_NAME_LENGTH))
+              .withResource(storageAccountId + "/blobServices/default")
+              .withLogAnalytics(logAnalyticsWorkspaceId)
+              .withLog("StorageRead", 0) // retention is handled by the log analytics workspace
+              .withLog("StorageWrite", 0)
+              .withLog("StorageDelete", 0);
+
+      var appInsights =
+          armManagers
+              .applicationInsightsManager()
+              .components()
+              .define(
+                  nameGenerator.nextName(
+                      ResourceNameGenerator.MAX_APP_INSIGHTS_COMPONENT_NAME_LENGTH))
+              .withRegion(resourceGroup.region())
+              .withExistingResourceGroup(resourceGroup.name())
+              .withKind("java")
+              .withApplicationType(ApplicationType.OTHER)
+              .withWorkspaceResourceId(logAnalyticsWorkspaceId);
+
       return deployment
           .withResourceWithPurpose(aks, ResourcePurpose.SHARED_RESOURCE)
           .withResourceWithPurpose(batch, ResourcePurpose.SHARED_RESOURCE)
-          .withResourceWithPurpose(storage, ResourcePurpose.SHARED_RESOURCE)
```

Comment:
Is the storage account still returned by the API as a landing zone "resource"?

Reply:
I think so, the storage account was moved the prerequisites but is still tagged correctly